.github/workflows/auto-generate.yml

```
@@ -0,0 +1,166 @@
+    - name: Setup Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '20'
+
+    - name: Install Dependencies
+      run: npm install
+
+    - name: Extract Tool Info from Issue (if applicable)
+      id: extract-info
+      if: github.event_name == 'issues'
+      run: |
```

actionlint (1.7.7)

"github.event.issue.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)